src/Backend/Modules/Location/Js/Location.js

```diff
@@ -141,38 +141,10 @@ export class Location {
     window.location = reloadLocation
   }
 
-  /**
-   * Save live data will save the setting in database
-   */
-  saveLiveData () {
-    $.ajax({
-      data: {
-        fork: {module: 'Location', action: 'SaveLiveLocation'},
-        zoom: this.zoomLevel,
-        type: this.type,
-        style: this.style,
-        centerLat: this.centerLat,
-        centerLng: this.centerLng,
-        height: this.height,
-        width: this.width,
-        id: this.mapId,
-        link: this.showLink,
-        directions: this.showDirections,
-        showOverview: this.showOverview
-      },
-      success: (json, textStatus) => {
-        // reload the page on success
-        if (json.code === 200) {
-          // no redirect given, refresh the page
-          if (typeof $('input#redirect').val() === 'undefined') {
-            this.refreshPage('map-saved')
-          }
-
-          $('input#redirect').val('edit')
-          $('form#edit').submit()
-        }
-      }
-    })
+  setCenter () {
+    this.getMapData()
+    $('#centerLat').val(this.centerLat)
```

Comment:
please use data attributes as selector in the JS.

Reply:
fixed bfb7b8e6ab81b58d0a824405378afac64cf475c0